core/inc/knoting/networking_types.h

```diff
@@ -0,0 +1,162 @@
+#pragma once
+#include <knoting/types.h>
+#include <yojimbo.h>
+
+namespace knot {
+using namespace yojimbo;
+
+constexpr int SERVER_PORT = 13189;
+constexpr int CLIENT_PORT = 12646;
+constexpr int MAX_CLIENTS = 6;
+const char* const SERVER_ADDRESS = "100.117.241.88";
+// const char* const SERVER_ADDRESS = "127.0.0.1";
+//^^^ FOR LOCAL HOST TESTING ^^^
+const double TICK = 1.0 / 30.0;
+
+struct GameConfig : public ClientServerConfig {
+    GameConfig() {
+        ClientServerConfig();
+        this->numChannels = 2;
+        this->channel[0].type = yojimbo::CHANNEL_TYPE_RELIABLE_ORDERED;
+        this->channel[1].type = yojimbo::CHANNEL_TYPE_UNRELIABLE_UNORDERED;
+        this->timeout = 5;
+        this->networkSimulator = false;
+        this->maxPacketSize = 1500;
```

Comment:
could consider timeout =5 and maxPacketSize = 1500
being variables if they are subject to check if not consider constexpr

Reply:
These will be const but it's setting of predefined variables within the Yojimbo system, I can declare the numbers as constexpr above and set the variables to these?

Reply:
Yep, please add them above as `constexpr`